build-tools/packages/build-cli/src/commands/info.ts

```diff
@@ -43,34 +95,53 @@ export default class InfoCommand extends BaseCommand<typeof InfoCommand> {
 				? context.packagesInReleaseGroup(flags.releaseGroup)
 				: [...context.fullPackageMap.values()];
 
+		// Sort by packages by name (invariant sort by codepoints).
+		// (See https://stackoverflow.com/a/40355107)
+		packages.sort(
+			(left, right) => Number(left.name > right.name) || -(left.name < right.name),
+		);
+
 		// Filter out private packages
 		if (!flags.private) {
 			packages = packages.filter((p) => p.packageJson.private !== true);
 		}
 
-		// eslint-disable-next-line import/no-deprecated
-		const tableData: (string | MonoRepoKind | undefined)[][] = [
-			["Release group", "Name", "Private", "Version"],
+		const columnNames = flags.columns;
+		const columnInfos = columnNames.map((name) => nameToColumnInfo[name]);
+
+		// Initialize 'tableData' with Pascal cased column names.
+		const tableData = [
+			columnNames.map((name) => {
+				return name.charAt(0).toUpperCase() + name.slice(1);
```

Comment:
Consider using changeCase, which is already a dependency. Would be self-documenting:

```ts
changeCase.PascalCase(name);
```

Reply:
Nice!  Thanks for the tip.